### PR TITLE
Closes #4541 Update tutorials categories and add new videos

### LINKED ISSUE #4541: Tutorials Tab - Reorganize and Add Videos
**Reorganize the videos in 4 categories (see screenshot attached):**

1. Getting Started
2. Passing the Core Web vitals
3. Troubleshooting
4. Add-ons

**Add 3 new videos for the Core Web Vitals category:**

- How to improve LCP with WP Rocket
`<script src="https://fast.wistia.com/embed/medias/fj7gs031q6.jsonp" async></script><script src="https://fast.wistia.com/assets/external/E-v1.js" async></script><div class="wistia_responsive_padding" style="padding:56.25% 0 0 0;position:relative;"><div class="wistia_responsive_wrapper" style="height:100%;left:0;position:absolute;top:0;width:100%;"><div class="wistia_embed wistia_async_fj7gs031q6 videoFoam=true" style="height:100%;position:relative;width:100%"><div class="wistia_swatch" style="height:100%;left:0;opacity:0;overflow:hidden;position:absolute;top:0;transition:opacity 200ms;width:100%;"></div></div></div></div>`

- How to improve FID with WP Rocket
`<script src="https://fast.wistia.com/embed/medias/krqb7ckugp.jsonp" async></script><script src="https://fast.wistia.com/assets/external/E-v1.js" async></script><div class="wistia_responsive_padding" style="padding:56.25% 0 0 0;position:relative;"><div class="wistia_responsive_wrapper" style="height:100%;left:0;position:absolute;top:0;width:100%;"><div class="wistia_embed wistia_async_krqb7ckugp videoFoam=true" style="height:100%;position:relative;width:100%"><div class="wistia_swatch" style="height:100%;left:0;opacity:0;overflow:hidden;position:absolute;top:0;transition:opacity 200ms;width:100%;"></div></div></div></div>`

- How to improve CLS with WP Rocket
`<script src="https://fast.wistia.com/embed/medias/5wrdzcmj90.jsonp" async></script><script src="https://fast.wistia.com/assets/external/E-v1.js" async></script><div class="wistia_responsive_padding" style="padding:56.25% 0 0 0;position:relative;"><div class="wistia_responsive_wrapper" style="height:100%;left:0;position:absolute;top:0;width:100%;"><div class="wistia_embed wistia_async_5wrdzcmj90 videoFoam=true" style="height:100%;position:relative;width:100%"><div class="wistia_swatch" style="height:100%;left:0;opacity:0;overflow:hidden;position:absolute;top:0;transition:opacity 200ms;width:100%;"></div></div></div></div>`

Tutorial tab

### PR DESCRIPTION
**Pull request by remyperona**
## Description

Update the tutorials categories, re-organize the videos and add the new ones about core web vitals

Fixes #4541

## Type of change

- [x] Enhancement (non-breaking change which improves an existing functionality)

## Is the solution different from the one proposed during the grooming?

No

## How Has This Been Tested?

- [x] Checked the tutorials page after the changes

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

### CODE REVIEW

File: views/settings/page-sections/tutorials.php [hunk 1]
```diff
@@ -16,20 +16,23 @@
...
 		],
 	],
 	[
-		'title'     => __( 'File Optimization', 'rocket' ),
+		'title'     => __( 'Passing the Core Web vitals', 'rocket' ),
 		'tutorials' => [
-			'frwm2xrksl' => __( 'Troubleshooting Display Issues with File Optimization', 'rocket' ),
-			'95z0cb0yxb' => __( 'How to Find the Right JavaScript to Exclude', 'rocket' ),
-			'9m1zg8p5wc' => __( 'How External Content Slows Your Site', 'rocket' ),
+			'fj7gs031q6' => __( 'How to improve LCP with WP Rocket', 'rocket' ),
```
Comment (iCaspar) [inline views/settings/page-sections/tutorials.php:25]: @Agathemed 
I know that the text here is what was specified in the issue, but for a new customer the abbreviations (LCP, FID, CLS) may not be familiar. Should they be spelled out?

**Review by iCaspar — APPROVED** [review]
From a code point of view this is all good. Just wondering about the abbreviations mentioned below.

File: views/settings/page-sections/tutorials.php (hunk 1 above)
Comment (iCaspar) [reply views/settings/page-sections/tutorials.php:25]: All good here!

**Comment by Mai-Saad** [comment]
@Tabrisrp Please find notes/Qs below while exploring the PR:

1. For Getting Started section, Can we adjust the cards so as not to have an empty card in between?
videos

2. Is it intended that controls for new videos are different from the old ones? , (I think better all be the same if doable, will vote for old ones controls)
old:
Screenshot from 2021-12-01 08-17-09
new:
Screenshot from 2021-12-01 08-17-51

3. Can we guard against this console error, it happens once?
Screenshot from 2021-12-01 08-16-14

4. On trunk as well: is it intended that the closing icon be outside the video area?

5. As a future enhancement: background images of videos better be homogenous (now old ones with text then dotted line below, new ones with icon & text below)

6. Note: Before release, we need to ensure having the translation for new videos labels and new category titles

7. On trunk: Just confirming that the same video (en version) is displayed independently on WP language

**Comment by Agathemed** [comment]
> Is it intended that controls for new videos are different from the old ones? , (I think better all be the same if doable, will vote for old ones controls)

It looks like Wistia is doing that automatically because of chapters in the video. There was no changes in the styles for controls.

> As a future enhancement: background images of videos better be homogenous (now old ones with text then dotted line below, new ones with icon & text below)

This is not new vs old, it's just a different thumbnail for Core Web Vitals videos. We are taking advantage of the illustration to explain the metric.

**Comment by remyperona** [comment]
1: I pushed a CSS change for that
2, 3, 4, 5: that's all handled on Wistia side, not ours
6: Yes
7: Yes

**Review by Mai-Saad — APPROVED** [review]
Working as expected

**Pull request merged**